recipes/default.rb

```diff
@@ -32,11 +32,32 @@
   mode 00755
 end
 
-execute 'install homebrew' do
-  command homebrew_go
-  environment lazy { { 'HOME' => ::Dir.home(homebrew_owner), 'USER' => homebrew_owner } }
-  user homebrew_owner
-  not_if { ::File.exist? '/usr/local/bin/brew' }
+begin
+  template '/etc/sudoers.d/homebrew_sudo' do
```

Comment:
I'm with you that it's older. However, chef-dk still supports 10.9. Plus the `metadata.rb` file doesn't have any macOS version restrictions currently.